### ISSUE TITLE
The system cannot find the file specified

### ISSUE THREAD

**Issue by premkgisl** [OpenRefine/OpenRefine#4415]
![image](https://user-images.githubusercontent.com/83585196/148719103-6214cde5-0333-4223-9ea3-fc7a790db627.png)

I got an error while generating the build. some target files missing(openrefine-mac-3.6-SNAPSHOT.dmg).

**Comment by wetneb** [comment]
Thanks for reporting this! This seems to be due to the fact that creating the Mac bundle is not supported on Windows, so this artifact should be skipped.

As a mitigation measure, you could avoid doing `mvn install` and simply run `mvn package`, unless you specifically need OpenRefine's jars to be available in your local Maven repository.

**Comment by wetneb** [comment]
Runing `mvn install` on Windows should now fail much later, at the packaging stage. Hopefully that already helps a little bit, since the interesting artifacts to install (the jars) should have already been installed at that stage.
Now it should just be a matter of skipping the attachment of the `dmg` artifact when run on Windows (since it can only be produced on Mac and Linux).

**Comment by frafra** [comment]
I can confirm that even by trying to build on Linux, it still tries to generate a DMG image. I used this command: `mvn -B package -P linux -DskipTests=true`.

**Comment by wetneb** [comment]
Yes, the `package` goal runs packaging for all three platforms (Windows, Linux, MacOS).
